client/capital/index.tsx

```diff
@@ -0,0 +1,211 @@
+import Page from 'components/page';
+import { TestModeNotice, topics } from 'components/test-mode-notice';
+import ErrorBoundary from 'components/error-boundary';
+import ActiveLoanSummary from 'components/active-loan-summary';
+import { formatExplicitCurrency, isZeroDecimalCurrency } from 'utils/currency';
+import { CapitalLoan } from 'data/capital/types';
+import ClickableCell from 'components/clickable-cell';
+import Chip from 'components/chip';
+import { useLoans } from 'wcpay/data';
+
+import './style.scss';
+
+const columns = [
+	{
+		key: 'paid_out_at',
+		label: __( 'Dispursed', 'woocommerce-payments' ),
+		screenReaderLabel: __( 'Dispursed', 'woocommerce-payments' ),
+		required: true,
+		isLeftAligned: true,
+		defaultSort: true,
+	},
+	{
+		key: 'status',
+		label: __( 'Status', 'woocommerce-payments' ),
+		screenReaderLabel: __( 'Status', 'woocommerce-payments' ),
+		required: true,
+		cellClassName: 'is-center-aligned',
+	},
+	{
+		key: 'amount',
+		label: __( 'Amount', 'woocommerce-payments' ),
+		screenReaderLabel: __( 'Amount', 'woocommerce-payments' ),
+		required: true,
+		isLeftAligned: false,
+	},
+	{
+		key: 'fee_amount',
+		label: __( 'Fixed fee', 'woocommerce-payments' ),
+		screenReaderLabel: __( 'Fixed fee', 'woocommerce-payments' ),
+		required: true,
+		isLeftAligned: false,
+	},
+	{
+		key: 'withhold_rate',
+		label: __( 'Withhold rate', 'woocommerce-payments' ),
+		screenReaderLabel: __( 'Withhold rate', 'woocommerce-payments' ),
+		required: true,
+		isLeftAligned: false,
+	},
+	{
+		key: 'first_paydown_at',
+		label: __( 'First paydown', 'woocommerce-payments' ),
+		screenReaderLabel: __( 'First paydown', 'woocommerce-payments' ),
+		required: true,
+		isLeftAligned: false,
+	},
+];
+
+const getLoanStatusText = ( loan: CapitalLoan ) => {
+	return loan.fully_paid_at
+		? __( 'Paid off', 'woocommerce-payments' ) +
+				': ' +
+				dateI18n( 'M j, Y', loan.fully_paid_at )
+		: __( 'Active', 'woocommerce-payments' );
+};
+
+const getLoanStatusChip = ( loan: CapitalLoan ) => {
+	return (
+		<Chip
+			message={ getLoanStatusText( loan ) }
+			type={ loan.fully_paid_at ? 'primary' : 'warning' }
+		/>
+	);
+};
+
+const getRowsData = ( loans: CapitalLoan[] ) =>
+	loans.map( ( loan ) => {
+		const clickable = ( children: React.ReactNode ) => (
+			<ClickableCell
+				href={
+					'/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Ftransactions' +
+					'&type=charge&filter=advanced&loan_id_is=' +
+					loan.stripe_loan_id
+				}
+			>
+				{ children }
+			</ClickableCell>
+		);
+
+		const data = {
+			paid_out_at: {
+				value: loan.paid_out_at,
+				display: clickable( dateI18n( 'M j, Y', loan.paid_out_at ) ),
+			},
+			status: {
+				value: getLoanStatusText( loan ),
+				display: clickable( getLoanStatusChip( loan ) ),
+			},
+			amount: {
+				value: isZeroDecimalCurrency( loan.currency )
+					? loan.amount
+					: loan.amount / 100,
+				display: clickable(
+					formatExplicitCurrency( loan.amount, loan.currency )
```

Comment:
I think the designs were done before we decided on a consistent approach for displaying money amounts. I looked at other similar tables (transactions, deposits) and they all use the explicit format. So that's what I went with.

Reply:
That makes sense, let's keep the explicit format then 🙂